SwiftSampleProject/SwiftyVokoder/Models/Station.swift

```diff
@@ -70,3 +70,5 @@ extension Station: VOKMappableModel {
         }
     }
 }
+
+extension Station: VokoderTypedManagedObject { }
```

Comment:
a) Needs moar `//MARK`
b) Since this is a sample project, is it worth throwing in any kind of note about the fact that this (and the several below) are basically mixin declarations?